cma/purecma.py

```diff
@@ -681,12 +692,12 @@ def legend_(*args, **kwargs):
         CMAESDataLogger.plotted += 1
 
     def save(self, name=None):
-        """Save data to file `name` or ``self.filename``"""
+        """Save data to file `name` or ``self.filename``."""
```

Comment:
I use the missing dot as a convention that this is all there is in the docstring.